scripts/projects_page/write_yaml.py

```diff
@@ -0,0 +1,469 @@
+        )
+
+        cursor = conn.cursor()
+        cursor.execute(query)
+        info = cursor.fetchall()
+
+    except mysql.connector.Error as err:
+        if err.errno == errorcode.ER_ACCESS_DENIED_ERROR:
+            print("Something is wrong with your user name or password")
+        elif err.errno == errorcode.ER_BAD_DB_ERROR:
+            print("Database does not exist on the server")
+        else:
+            print(err)
+
+    cursor.close()
+    conn.close()
+    return info
+
+
+def write_yaml(info_dict, icon, yaml_out, project, use_server):
+    # there are species on main for which the upper case production name is used in the url instead of the upper case species name
+    prod_url_list = ["bos_taurus_hybrid", "bos_indicus_hybrid"]
+
+    assembly_name = info_dict["assembly.name"].replace(" ", "_")
+    date = info_dict["genebuild.last_geneset_update"].replace("-", "_")
+    species_name = info_dict["species.scientific_name"].replace(" ", "_")
+    if "species.strain" in info_dict:
+        if info_dict["species.strain"] != "reference":
+            info_dict["species.scientific_name"] = (
+                info_dict["species.scientific_name"]
+                + " ("
+                + info_dict["species.strain"]
+                + ")"
+            )
+    lc_species_name = (info_dict["species.scientific_name"].replace(" ", "_")).lower()
+    uc_prod_name = (info_dict["species.production_name"]).capitalize()
+
+    assembly_report_url = (
+        "https://www.ncbi.nlm.nih.gov/assembly/" + info_dict["assembly.accession"]
+    )
+    assembly_report_response = requests.get(assembly_report_url)
+    submitter_match = re.search(
+        "Submitter: </dt><dd>([^<]*)</dd><dt>", assembly_report_response.text
+    )
+    if submitter_match:
+        submitter = submitter_match.group(1)
+    else:
+        submitter = "unknown"
+    # for the chicken reference the submitter info is missing, set it manually
+    if info_dict["assembly.accession"] == 'GCA_000002315.5':
+        submitter = "Genome Reference Consortium"
+
+    if use_server == "rapid":
+        ftp_base = "http://ftp.ensembl.org/pub/rapid-release/species"
+
+        yaml = "- species: " + info_dict["species.scientific_name"] + "\n"
+
+        if project in ("vgp", "dtol"):
+            yaml += "  image: " + icon + "\n"
+        else:
+            yaml += "  submitted_by: " + submitter + "\n"
+
+        yaml += "  accession: " + info_dict["assembly.accession"] + "\n"
+        yaml += (
+            "  annotation_gtf: "
+            + ftp_base
+            + "/"
+            + species_name
+            + "/"
+            + info_dict["assembly.accession"]
+            + "/geneset/"
+            + date
+            + "/"
+            + species_name
+            + "-"
+            + info_dict["assembly.accession"]
+            + "-"
+            + date
+            + "-genes.gff3.gz\n"
+        )
```

Comment:
f-string:
```
        yaml += f'  annotation_gtf: {ftp_base}/{species_name}/{info_dict["assembly.accession"]}/geneset/{date}/{species_name}-{info_dict["assembly.accession"]}-{date}-genes.gff3.gz\n'
```

string format method:
```
        yaml += "  annotation_gtf: {}/{}/{}/geneset/{}/{}-{}-{}-genes.gff3.gz\n".format(
            ftp_base,
            species_name,
            info_dict["assembly.accession"],
            date,
            species_name,
            info_dict["assembly.accession"],
            date,
        )
```

I think the second one looks a lot better in this case.